lib/infoblox/resource/aaaa_record.rb

```diff
@@ -0,0 +1,9 @@
+# An AAAA (address) record maps a domain name to an IPv6 address.
+module Infoblox
+  class AAAArecord < Resource
+    remote_attr_accessor :name, :ipv6addr, :ttl,
+                         :extattr, :view, :comment, :disable, :dns_name, :use_ttl, :zone
```

Comment:
I assume this is missing extensible_attributes because `AAAArecord` is not available to versions that also use `extensible_attributes`.  Is that the case?  Perhaps a comment is warranted.